NAMESPACE

```diff
@@ -70,14 +73,17 @@ export(sqlParseVariables)
 export(sqlParseVariablesImpl)
 export(sqlQuoteSpec)
 export(sqlRownamesToColumn)
+exportClasses(DBIArrowResult)
 exportClasses(DBIConnection)
 exportClasses(DBIConnector)
 exportClasses(DBIDriver)
 exportClasses(DBIObject)
 exportClasses(DBIResult)
+exportClasses(DBIResultArrow)
```

Comment:
Do we need both?

Reply:
Mostly down to poor naming: one is the "abstract" class to be overridden by backends and the other extends this and provides a default implementation. Happy to take suggestions. `DBIResultArrowDefault`?

Reply:
We now have `DBIResultArrow` and `DBIResultArrowDefault`.